ert3/data/_record.py

```diff
@@ -3,11 +3,12 @@
 
 
 class _DataElement(BaseModel):
-    validate_all = True
-    validate_assignment = True
-    extra = "forbid"
-    allow_mutation = False
-    arbitrary_types_allowed = True
+    class Config:
```

Comment:
Care to elaborate on this? Is the Pydantic syntax wrong as is?

Reply:
Yes. Currently, you're not making a global configuration, but adding these fields onto every single record. You meant to put it in a `class Config:`

https://pydantic-docs.helpmanual.io/usage/model_config/

Reply:
Excellent 👏 I suggest we move this fix into a separate PR with a failing test and the fix in two consequative commits 👍